kasa/cli/main.py

```diff
@@ -347,6 +336,19 @@ async def cli(
         )
         dev = await Device.connect(config=config)
         device_updated = True
+    elif alias:
+        echo(f"Alias is given, using discovery to find host {alias}")
+
+        from .discover import find_dev_from_alias
+
+        dev = await find_dev_from_alias(
+            alias=alias, target=target, credentials=credentials
+        )
+        if not dev:
+            echo(f"No device with name {alias} found")
+            return
+        echo(f"Found hostname by alias: {host}")
```

Comment:
The host should be read from the dev, right?

Reply:
Fixed